backend/src/akvo/lumen/lib/import/flow_common.clj

```diff
@@ -1,53 +1,66 @@
 (ns akvo.lumen.lib.import.flow-common
-  (:require [akvo.commons.psql-util :as pg]
-            [cheshire.core :as json]
-            [akvo.lumen.http.client :as http.client]
-            [clojure.java.jdbc :as jdbc]
-            [clojure.string :as str])
+  (:require
+   [akvo.commons.psql-util :as pg]
+   [akvo.lumen.http.client :as http.client]
+   [cheshire.core :as json]
+   [clojure.java.jdbc :as jdbc]
+   [clojure.string :as str]
+   [clojure.tools.logging :as log]
+   [diehard.core :as dh])
```

Comment:
Why this re-indentation is required?

Reply:
It's not, it happened while playing with diehard and pulling in a temporary http client.